Week1/app/src/main/java/com/example/week1/db/TodoViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.week1.db
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+
+class TodoViewModel(
+    todoDao: TodoDao
+) : ViewModel() {
+
+    private val repository: TodoRepository = TodoRepository(todoDao)
```

Comment:
```suggestion
class TodoViewModel(
    private val repository: TodoRepository
) : ViewModel() {    
```

UI Layer에서 dao 접근하는 것은 지양하는 것이 좋을 갓 같습니다. 이유를 한번 생각해주시겠어요?